cycles-ledger/src/main.rs

```diff
@@ -40,9 +40,9 @@ fn init(ledger_args: LedgerArgs) {
 fn post_upgrade(ledger_args: Option<LedgerArgs>) {
     match ledger_args {
         Some(LedgerArgs::Upgrade(Some(upgrade_args))) => {
-            if let Some(max_transactions_per_request) = upgrade_args.max_transactions_per_request {
+            if let Some(max_transactions_per_request) = upgrade_args.max_blocks_per_request {
```

Comment:
`max_transactions_per_request` should be renamed too, right?

Reply:
ahh missed one. Thanks!